src/System.IO.Compression.Brotli/System/Interop/Library.Windows.cs

```diff
@@ -6,7 +6,7 @@ namespace System.IO.Compression
 {
     internal static partial class Library
     {
-        internal const string BrotliEnc = "brotlienc.dll";
-        internal const string BrotliDec = "brotlidec.dll";
+        internal const string _brotliEncoder = "brotlienc.dll";
```

Comment:
You only changed Library.Windows.cs to _brotliEncoder/etc.
Please also make the same change to:
- Library.OSX.cs
- Library.Unix.cs